data_structures/bloom_filters/bloom_filter/main.c

```diff
@@ -0,0 +1,21 @@
+#include "./bloom_filter.h"
+#include <sys/resource.h>
+
+int main(void) {
```

Comment:
[The contribution guideline](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#test) requires that you put these testing code in a separate test(), but IMHO this is fine. Your call